src/multiform/MultiForm.ts

```diff
@@ -134,13 +135,17 @@ export default class MultiForm extends AVisInstance implements IVisInstance, IMu
         };
         this.actVis.on('transform', t);
         const r = this.actVis.transform(scale, rotate);
+        this.options.all.heightTo = this.actVis.rawSize[1] * (scale[1]||1);
```

Comment:
this sounds like a hack, which may only work for mothertable but not in general. In addition, why is `this.actVis.size[1]` not enough?